app/graphql/queries/fetch_user_month_footprint.rb

```diff
@@ -0,0 +1,13 @@
+module Queries
+  class FetchUserMonthFootprint < Queries::BaseQuery
+    type Types::CarMonthlyMileage, null: true
```

Comment:
I think the name for this should be changed to FetchUserCarMonthFootprint in case we add future endpoints with flights, etc. A UserMonthFootprint could be an endpoint that aggregates all month footprint types.

Reply:
Good thought, just pushed this change up 👍

Reply:
Noice